### PR TITLE
Update documentation reflect web app config side panel

### PR DESCRIPTION
**Pull request by Thushani-Jayasekera**
>To make it easier, we want to create a user-friendly interface, like the one used for BYOC deployment setup. This would let web app users simply upload or edit a configuration file.

Resolves issue https://github.com/wso2-enterprise/choreo/issues/23529

Relates to the PR https://github.com/wso2-enterprise/choreo-console/pull/7926

https://github.com/wso2-enterprise/choreo-console/assets/62471034/efc1aa6a-2738-42e9-817f-d41755016b09

image

### CODE REVIEW

File: en/docs/develop-components/develop-a-web-application.md [hunk 1]
```diff
@@ -26,6 +26,57 @@ You can create a Web Application in Choreo as follows:
 
     Once you create the Web Application component, Choreo will automatically generate a build pipeline for your Single Page Application and deploy it.
 
+    Follow the steps below to manage runtime configurations for your React application at manual deployment.
```
Comment (shaniR) [inline en/docs/develop-components/develop-a-web-application.md:29]: Is this applicable only for React?

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md [hunk 2]
```diff
@@ -212,9 +212,10 @@ Let's deploy a front-end application to consume the API. This application is des
 Once you create the web application component, you can deploy it to the Choreo runtime. To deploy the web application component, follow the steps below:
 
 1. In the left menu, click **Deploy**.
-2. In the **Build Area** card, click **Build and Deploy**. The deployment may take a few minutes to complete.
-3. Once you deploy the web application, copy the **Web App URL** from the development environment card.
-4. Navigate to the web app URL. You can verify that you have successfully hosted the web application.
+2. In the **Build Area** card, click **Configure & Deploy**. 
+3. Add configurations and deploy. The deployment may take a few minutes to complete. See how to define and read configurations at [Develop a Web Application page](https://wso2.com/choreo/docs/develop-components/develop-a-web-application/#creating-a-web-application).
```
Comment (shaniR) [inline en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:216]: What configurations are these? And what values do users have to add?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md (hunk 2 above)
Comment (Thushani-Jayasekera) [reply en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:216]: Please refer this comment to view document changes request - https://github.com/wso2-enterprise/choreo/issues/23014#issuecomment-1689624405

File: en/docs/develop-components/develop-a-web-application.md (hunk 1 above)
Comment (Thushani-Jayasekera) [reply en/docs/develop-components/develop-a-web-application.md:29]: I will adjust it. should be available for all SPA which need to apply a file mount at the time of deployment.

**Comment by Thushani-Jayasekera** [comment]
Since there were multiple features/changes that were pushed to Choreo recently, the QSG will undergo multiple changes. Therefore, required will be updated separately.

**Pull request closed without merging**